src/main/java/com/checkmarx/ast/wrapper/CxWrapper.java

```diff
@@ -98,6 +98,21 @@ public Scan scanCreate(@NonNull Map<String, String> params, String additionalPar
             throws IOException, InterruptedException, CxException {
         this.logger.info("Executing 'scan create' command using the CLI.");
 
+        List<String> arguments = buildScanCreateArgumentsArray(params, additionalParameters);
+
+        return Execution.executeCommand(withConfigArguments(arguments), logger, Scan::fromLine);
+    }
+
+    //Used in Jenkins plugin
```

Comment:
Let's keep wrapper generic, no need to specify jenkins.
scanCreate could use scanBuild?
perhaps scanBuild should be more specifically named like: buildScanCreateArguments